spring-r2dbc/src/main/java/org/springframework/r2dbc/connection/TransactionAwareConnectionFactoryProxy.java

```diff
@@ -58,7 +58,7 @@
  * {@link ConnectionFactory}, avoiding the need to define such a proxy in the first place.
  *
  * <p><b>NOTE:</b> This {@link ConnectionFactory} proxy needs to return wrapped
- * {@link Connection}s (which implement the {@link ConnectionProxy} interface) in order
+ * {@link Connection}s (which implement the {@link org.springframework.jdbc.datasource.ConnectionProxy} interface) in order
```

Comment:
Please revert this change.

This is obsolete documentation which needs to be reworked separately.